src/pages/en/guides/imports.md

```diff
@@ -35,23 +63,23 @@ import { getUser } from './user.js';
 
 All browsers now support ESM, so Astro is able to ship this code directly to the browser during development.
```

Comment:
```suggestion
ESM is now native to all browsers, this lets Astro ship code directly to the browser during development, using Astro's own Hot-Module Reloading (HMR).
```

Reply:
Changed! Better, thanks! (But, it's 2 diff sentences, so I just edited myself.)